clang/lib/Sema/Sema.cpp

```diff
@@ -707,6 +707,7 @@ ExprResult Sema::ImpCastExprToType(Expr *E, QualType Ty,
     case CK_ToVoid:
     case CK_NonAtomicToAtomic:
     case CK_HLSLArrayRValue:
+    case CK_HLSLAggregateCast:
```

Comment:
@llvm-beanz  Should this cast be here?

Reply:
Nope, this cast should not ever be implicit.